cmd/oci-runtime-tool/generate.go

```diff
@@ -608,6 +613,18 @@ func parseRlimit(rlimit string) (string, uint64, uint64, error) {
 	return parts[0], uint64(hard), uint64(soft), nil
 }
 
+func parseNamespace(ns string) (string, string, error) {
+	parts := strings.SplitN(ns, ":", 2)
```

Comment:
There aren't any unit tests or integration tests for `oci-runtime-tool`. We should probably add some. Also, this tool isn't versioned which makes [packaging annoying](https://build.opensuse.org/package/show/Virtualization:containers/oci-runtime-tools).